test/integration/spec/localparticipant.js

```diff
@@ -1172,6 +1219,45 @@ describe('LocalParticipant', function() {
     });
   });
 
+  (defaults.topology === 'peer-to-peer' ? describe.skip : describe)('JSDK-2534', () => {
+    let localVideoTrackPublications;
+    let localVideoTracks;
+    let room;
+    let sid;
+
+    before(async () => {
+      [sid, ...localVideoTracks] = await Promise.all([
+        createRoom(randomName(), defaults.topology)
+      ].concat([
+        smallVideoConstraints,
+        smallVideoConstraints
+      ].map(createLocalVideoTrack)));
```

Comment:
can this be easier with:
```
      sid = await createRoom(randomName(), defaults.topology);
      localVideoTracks = await Promise.all([smallVideoConstraints, smallVideoConstraints].map(createLocalVideoTrack));

```

Reply:
it does take a bit longer, as we are not waiting for all promises simultaneously, but I feel the benefit of readability + smaller code is worth it especially give its test code.

Reply:
btw. I found that when tests do timeout, it's much easier to see what went wrong by using 
```
      const waitFor = require('../../lib/util').waitFor;
      localVideoTracks = await waitFor([smallVideoConstraints, smallVideoConstraints].map(createLocalVideoTrack), "waiting for video tracks");
```
but maybe `waitFor` needs a tweak to defaults to 1) use a smaller timeout 2) not be so verbose by default.